sound/soc/intel/boards/bytcht_da7213.c

```diff
@@ -43,6 +43,10 @@ static const struct snd_soc_dapm_widget dapm_widgets[] = {
 	SND_SOC_DAPM_MIC("Headset Mic", NULL),
 	SND_SOC_DAPM_MIC("Mic", NULL),
 	SND_SOC_DAPM_LINE("Aux In", NULL),
+	SST_AIF_IN("codec_in0", NULL),
+	SST_AIF_IN("codec_in1", NULL),
+	SST_AIF_OUT("codec_out0", NULL),
+	SST_AIF_OUT("codec_out1", NULL),
```

Comment:
NAK? It looks like a complete misunderstanding, the ask was to create virtual widgets in the topology file e.g. https://github.com/thesofproject/sof/blob/a2e6f82ef810a6a838783467472dfceb7933afc1/tools/topology/sof-apl-da7219.m4#L174